requirements-test.in

```diff
@@ -31,4 +31,6 @@ lm-eval[api]==0.4.4 # required for model evaluation test
 bitsandbytes>=0.44.0
 buildkite-test-collector==0.1.9
 
+genai_perf==0.0.8
```

Comment:
@khluu @youkaichao just want to double check and see if it is OK to introduce new dependencies in `requirements-test.in`. Other parts of the code looks good to me.

Reply:
SGTM. It just needs to be compiled with Python 3.12

Reply:
SGTM.